lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -249,6 +252,22 @@ private QuestionRequest createQuestionRequest(PersonIdentityDetailed personIdent
         return questionRequest;
     }
 
+    private String retrieveQuestionStrategy()
+            throws JsonProcessingException, InvalidStrategyScoreException {
+        var verificationScore = "2";
```

Comment:
Maybe make this a constant

Reply:
Maybe, could do something like `String strategy = strategyMap.getOrDefault(sessionVerificationScore. verificationScore);` will be a placeholder for the when we have something coming from the sessionItem